js/tooltip.js

```diff
@@ -238,6 +238,17 @@
     offset.top  = offset.top  + marginTop
     offset.left = offset.left + marginLeft
 
+    var screenWidth = $(window).width();
+    var screenHeight = $(window).height();
+    var resizeWidth = false;
```

Comment:
This can be simplified to:
```js
var resizeWidth  = (offset.left + width  > $(window).width());
var resizeHeight = (offset.top  + height > $(window).height());
```